notify/impl.go

```diff
@@ -141,13 +142,13 @@ var userAgentHeader = fmt.Sprintf("Alertmanager/%s", version.Version)
 // Webhook implements a Notifier for generic webhooks.
 type Webhook struct {
 	// The URL to which notifications are sent.
```

Comment:
Not sure why the URL was exported before, I moved the `Config` -> `conf` to be consistent with all the other implementations.